### PR TITLE
Thai translations

### PR DESCRIPTION
**Pull request by karidoe**
Hello,

I've finished translating the CI4 framework to the Thai language and would like to merge it back to the base repo. I've updated the `README.md` file also. But not sure if I gave the right version to it; I put the data in the `version` column as `4.0.3`. Feel free to give me feedback in case I did anything wrong.

### CODE REVIEW

File: tests/Language/AbstractTranslationTestCase.php [hunk 1]
```diff
@@ -65,6 +65,7 @@ abstract class AbstractTranslationTestCase extends TestCase
 		VietnameseTranslationTest::class         => 'vi',
 		SimpleChineseTranslationTest::class      => 'zh-CN',
 		TraditionalChineseTranslationTest::class => 'zh-TW',
+		ThaiTranslationsTest::class              => 'th',
```
Comment (paulbalandan) [inline tests/Language/AbstractTranslationTestCase.php:68]: Sort this by value, not by key.

File: README.md [hunk 2]
```diff
@@ -55,6 +55,7 @@ claims no expertise beyond English.
 | si     | Sinhala              | 4.0.3   | [npwsamarasinghe][si]| Nalaka Prasad
 | sk     | Slovak               | 4.0.0   | [xbotkaj][sk]        | Jozef Botka
 | sv-SE  | Swedish - Sweden     | 4.0.3   | [tangix][sv-SE]      | Mattias Sandström
+| th     | Thai                 | 4.0.3   | [karidoe][th]        | Anuchit Thiamuan
```
Comment (paulbalandan) [inline README.md:58]: Please change the version origin to 4.x to denote this is still unreleased. I'll change this later when we have a release.

File: tests/Language/ThaiTranslationsTest.php [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+ * For the full copyright and license information, please view
+ * the LICENSE file that was distributed with this source code.
+ */
+
+namespace Translations\Tests;
+
+/**
+ * @internal
+ */
+final class ThaiTranslationsTest extends AbstractTranslationTestCase
```
Comment (paulbalandan) [inline tests/Language/ThaiTranslationsTest.php:17]: Should be ThaiTranslationTest.

**Review by paulbalandan — COMMENTED** [review]
Hi! Thank you for this great addition. I just have some comments and then this is ready to merge.

**Comment by paulbalandan** [comment]
Please squash also your commits later. If you cannot do that, please tell me so I can do the squash during merge.

File: README.md (hunk 2 above)
Comment (karidoe) [reply README.md:58]: got it.

**Comment by karidoe** [comment]
> Please squash also your commits later. If you cannot do that, please tell me so I can do the squash during merge.
Yes, please. I need your help to squash commits for me this time. Thanks!

**Review by paulbalandan — APPROVED** [review]

**Comment by paulbalandan** [comment]
> > Please squash also your commits later. If you cannot do that, please tell me so I can do the squash during merge.
> 
> Yes, please. I need your help to squash commits for me this time. Thanks!

You definitely need to learn on interactive rebases. That can save you from a lot of commits. I'll take this from here. You can do the squashing on your next PRs.

**Comment by paulbalandan** [comment]
Thank you, @karidoe

**Pull request merged**